components/announcement_bar/configuration_bar/configuration_bar.jsx

```diff
@@ -95,6 +116,32 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
                     />
                 );
             }
+            if (this.props.license.IsLicensed === 'false' && this.props.warnMetricsStatus) {
+                for (const [id, flag] of Object.entries(this.props.warnMetricsStatus)) {
+                    if (flag) {
```

Comment:
can we reduce a bit the `hadouken` code here doing it like
```js
if (!flag) {
    continue
}
var notice = this.getNoticeForWarnMetricId(id);
if (notice.IsDismissed) {
    continue
}
return (...
```

Reply:
I don't think it rises to the level of hakouten code (otherwise we would call that any code that has more than one/two levels of wrapped ifs as such and I think that is a reach), but don't have strong preferences, so I'll split it vertically vs. horizontally = ).